### ISSUE TITLE
Failed: HostConnectionPoolSpec

### ISSUE THREAD

**Issue by chbatey** [akka/akka-http#1610]
https://jenkins.akka.io:8498/job/akka-http-nightly-against-akka-master/375/consoleFull

```
[info] HostConnectionPoolSpec:
[info] NewPoolImplementation on PassThrough
[info] - should complete a simple request/response cycle with a strict request and response *** FAILED ***
[info]   java.lang.AssertionError: expected OnNext, found OnComplete
[info]   at akka.stream.testkit.TestSubscriber$ManualProbe.expectNext(StreamTestKit.scala:327)
[info]   at akka.stream.testkit.TestSubscriber$ManualProbe.expectNext(StreamTestKit.scala:316)
[info]   at akka.stream.testkit.TestSubscriber$Probe.requestNext(StreamTestKit.scala:753)
[info]   at akka.http.impl.engine.client.HostConnectionPoolSpec$$anonfun$testSet$1$SetupWithServerProbes$2$ServerConnection.expectRequest(HostConnectionPoolSpec.scala:370)
[info]   at akka.http.impl.engine.client.HostConnectionPoolSpec$$anonfun$testSet$1$$anonfun$apply$mcV$sp$1$$anon$1.<init>(HostConnectionPoolSpec.scala:89)
[info]   at akka.http.impl.engine.client.HostConnectionPoolSpec$$anonfun$testSet$1$$anonfun$apply$mcV$sp$1.apply(HostConnectionPoolSpec.scala:85)
[info]   at akka.http.impl.engine.client.HostConnectionPoolSpec$$anonfun$testSet$1$$anonfun$apply$mcV$sp$1.apply(HostConnectionPoolSpec.scala:85)
[info]   at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]   ...
[info] - should complete a simple request/response cycle with a chunked request and response *** FAILED ***
```

**Comment by chbatey** [comment]
https://jenkins.akka.io:8498/job/akka-http-nightly-against-akka-master-on-2.12/359//consoleFull

**Comment by raboof** [comment]
Possibly https://jenkins.akka.io:8498/job/akka-http-nightly-against-akka-master-on-2.12/367//consoleFull

**Comment by jrudolph** [comment]
Fixed by #1625.

**Comment by raboof** [comment]
Possibly https://jenkins.akka.io:8498/job/akka-http-nightly-against-akka-master/384/consoleFull
